### ISSUE TITLE
Account-switcher modal

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-flutter#1038]
This is a follow-up to:
* #1037 

Before this issue (and after #1037), when the user chooses the "Organizations" option from the main menu, we take them to a choose-account page with a list of accounts.

This issue is to instead show a modal menu with a list of accounts, like this:
![image](https://github.com/user-attachments/assets/dc3a9304-45f1-46c8-9443-41da327a1083)

This doesn't yet have a fully-updated design, so it will need that before it's ready for implementation.

This also has a post-launch issue as a prerequisite, in order to have all the needed data:
* #1036 

So this is a post-launch issue too.

**Comment by yashrajjain726** [comment]
Hi @gnprice , I donot see this UI on the app that you posted screenshot with. Can you please elaborate more.

**Comment by chrisbobbe** [comment]
Hi @yashrajjain726, welcome! This issue isn't yet ready to work on. If you'd like to choose an issue to work on, please see our [guide](https://zulip.readthedocs.io/en/latest/contributing/contributing.html#in-other-zulip-repositories) to doing so and please choose a different issue.

**Comment by gnprice** [comment]
The screenshot in this issue description covers several changes which are independent of each other:
* Showing the unread count for each account. This has its own issue:
  #1160 
* Showing the realm's icon and name, rather than its URL and your email address. I've just filed an issue for that:
  #1943
* Changing the button at the bottom to have a "log in" icon and say something like "Log into another organization", instead of "Add an account" as it does today.
* Putting the whole list into a modal dialog instead of a whole page, and making other visual changes.

Last week we moved this issue up to the current milestone — but I think what we really meant was the realm icons and names, #1943. So I've put that issue in the current milestone and will update this one accordingly.

**Comment by gnprice** [comment]
One other change covered (I think) by the screenshot in this issue description, which I hadn't noticed until just now:
* Highlighting the current account. This was requested in user feedback yesterday:
  #2042 

 … On a closer look, I'm not entirely sure that's the intention in the design. The highlighted account in the screenshot doesn't match the account shown in the background. But I don't see what else would be intended as driving the highlight, so my guess is that that's just a small error in the design mockup.
